project/plugins.sbt

```diff
@@ -10,7 +10,7 @@ addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "1.15.0")
 
 addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "1.3.0")
 
```

Comment:
A bunch of plugins in here can be updated.

Reply:
This is checked using ```sbt dependencyUpdates``` rather than ```sbt validate```
https://github.com/rtimush/sbt-updates